src/Command/QueueCleanupCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Webgriffe\SyliusAkeneoPlugin\Command;
+
+use DateInterval;
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ManagerRegistry;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webgriffe\SyliusAkeneoPlugin\DateTimeBuilder;
+use Webgriffe\SyliusAkeneoPlugin\Entity\QueueItem;
+use Webgriffe\SyliusAkeneoPlugin\Repository\QueueItemRepositoryInterface;
+
+final class QueueCleanupCommand extends Command
+{
+    public const SUCCESS = 0;
+
+    public const FAILURE = 1;
+
+    public const DEFAULT_DAYS = 10;
+
+    public const DAYS_OPTION_NAME = 'days';
+
+    /** @var QueueItemRepositoryInterface */
+    private $queueItemRepository;
+
+    /** @var ManagerRegistry */
+    private $managerRegistry;
+
+    // the name of the command (the part after "bin/console")
+    protected static $defaultName = 'webgriffe:akeneo:cleanup-queue';
+
+    /**
+     * QueueCleanupCommand constructor.
+     */
+    public function __construct(QueueItemRepositoryInterface $queueItemRepository, ManagerRegistry $managerRegistry)
+    {
+        $this->queueItemRepository = $queueItemRepository;
+        $this->managerRegistry = $managerRegistry;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Clean the Akeneo\'s queue of items older than N days.')
+            ->setHelp('This command allows you to clean the Akeneo\'s queue of item older than a specificed numbers of days.')
+            ->addArgument(
+                self::DAYS_OPTION_NAME,
+                InputArgument::REQUIRED,
+                'Number of days from which to purge the queue of previous items',
+                (string) (self::DEFAULT_DAYS)
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        // get the number of days from user
+        $numberOfDays = $input->getOption(self::DAYS_OPTION_NAME);
```

Comment:
Did you tested it after changing to an argument? If I'm not wrong this should be `getArgument()` and not `getOption()`.